c2cgeoportal/lib/email_.py

```diff
@@ -34,23 +34,23 @@
 from email.Utils import formatdate
 
 
-def send_email(from_addr, to_addrs, body, subject, smtp):  # pragma: no cover
+def send_email(email_config, smtp_config):  # pragma: no cover
```

Comment:
Why do you replace `from_addr, to_addrs, body, subject` by `email_config`?

Reply:
since we pass the smtp config as an object, I thought it would be more coherent to also pass the email config as an object, 
but I can revert that if you dont like it